brave/browser-laptop
# Replace hideLower with showLess

Based on the discussion on Slack.

Also:
- Rename constants to sort them not only alphabetically but semantically (just sorting alphabetically breaks the semantic order of the existing constants)
- Add lacking states to docs/state.md

Closes #10158

Auditors: 

Test Plan:
1. Run `npm run add-simulated-synopsis-visits 100`
2. Open about:preferences#payments
3. Click `Show All` button
4. Click `Show Less` button
5. Make sure the buttons work

Submitter Checklist:


Test Plan:


Reviewer Checklist:

Tests
